README.md

```diff
@@ -257,6 +265,39 @@ jobs:
           flake8: true
 ```
 
+### C, C++, Objective-C and Objective-C++ example (clang_format)
+
+```yml
+name: Lint
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install ClangFormat
+        run: sudo apt-get install -y clang-format
+
+      - name: Run linters
+        uses: wearerequired/lint-action@v3
```

Comment:
Since current version is 2 and it does not support ClangFormat, I wrote `lint-action@v3` here. Tell me if it is not appropriate.

Reply:
You can keep this as v2 since adding a new format isn't a breaking change.

Reply:
Fixed in 1ceae759aa0f32e1c69777e4861cd5e4d08e755b